tenseal/proto/tensors.proto

```diff
@@ -9,6 +9,16 @@ message BFVVectorProto {
     repeated bytes ciphertexts = 2;
 };
 
+//BFVTensor parameters
+message BFVTensorProto {
+    // The shape of the encrypted tensor
+    repeated uint32 shape = 1;
+    // The serialized ciphertexts
+    repeated bytes ciphertexts = 2;
+    // Optional batch size. Exists only if batching is enabled
+    uint32 batch_size = 4;
```

Comment:
Why 4? this can be index 3.

Reply:
I should stop copy-pasting to be faster and forgetting about it, sorry.
I deleted the scale which was not needed but forgot indeed to change the index to 3.